msg/RoverDifferentialSetpoint.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp # time since system start (microseconds)
+
+float32 forward_speed_setpoint # [m/s or normalized [-1, 1]] Desired forward speed for the rover
```

Comment:
>[m/s or normalized [-1, 1]]

What speaks against separate fields? I don't think a field with different interpretations is good.

Reply:
Implemented [here](https://github.com/PX4/PX4-Autopilot/pull/23629/commits/2c76eb214a78a493dc9e1aac76e988cf7a0ec2b3#diff-fdc56dd532fa90878add2ff88f33a2a203c15e43afa376b1105c7a466e684366R3-R7).